imports/ui/Base.js

```diff
@@ -1,5 +1,6 @@
 import { Template } from 'meteor/templating';
 
-import './Body.html';
-
+import './Base.html';
```

Comment:
Why the rename?

Reply:
The "body" of the page should be anything that's not the navigation or the title, so I felt like using Base represents it better. Since you don't use it by itself, it serves as the base for every other route.